### PR TITLE
MDS-3602: Show the NoW/AA Number When Header is Fixed

### PR DESCRIPTION
**Pull request by exzizt**
# Main
* Shows the Notice of Work or Administrative Amendment number on the dashboard when the header is in fixed mode
image

### CODE REVIEW

File: services/core-web/src/components/noticeOfWork/applications/NOWTabHeader.js [hunk 1]
```diff
@@ -44,6 +46,15 @@ export const NOWTabHeader = (props) => (
...
               {props.tabName}
             </Popover>
+            {props.fixedTop && (
+              <div className="view--header--sub">
+                {props.noticeOfWork?.application_type_code == "NOW"
+                  ? "Notice of Work"
+                  : "Administrative Amendment"}
+                :
+                <br />#{props.noticeOfWork.now_number}
+              </div>
```
Comment (mmumby) [inline services/core-web/src/components/noticeOfWork/applications/NOWTabHeader.js:57]: can you add a default for those that have null now_numbers

Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/applications/NOWTabHeader.js:57]: Ideally, this field shouldn't be null, but there are close to 400 in test with null values. We should look into this...

Comment (exzizt) [reply services/core-web/src/components/noticeOfWork/applications/NOWTabHeader.js:57]: Should we display "N/A"? Or just remove this altogether if it is null?

Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/applications/NOWTabHeader.js:57]: I vote for "N/A", the same as the main header.

**Review by mmumby — APPROVED** [review]

**Pull request merged**